roles/openshift_logging_elasticsearch/templates/es.j2

```diff
@@ -58,6 +58,9 @@ spec:
               name: "cluster"
           env:
             -
+              name: "DC_NAME"
+              value: "{{deploy_name}}"
```

Comment:
I was unable to find a way to get this from downward api.  Fails with message that labels are not supported.  Depending on what version we wish to support, it also may not support annotations that include escape characters